cluster/addons/metrics-server/metrics-server-deployment.yaml

```diff
@@ -81,15 +81,16 @@ spec:
         command:
           - /pod_nanny
           - --config-dir=/etc/config
-          - --cpu=40m
+          - --cpu={{ base_metrics_server_cpu }}
           - --extra-cpu=0.5m
```

Comment:
For consistency I'd templetize cpu as well - all settings will be in a single place.

Reply:
The implementation of this change mirrors the implementation currently used for heapster. In both cases the extra-cpu flag is not being tuned. For consistency with existing code and since it is not critical to the functionality I'm looking to implement I chose not to templatize the value of that flag. I can, however, implement a follow-up change to do the proposed clean-up if desired.